scripts/policy/frameworks/cluster/experimental.zeek

```diff
@@ -0,0 +1,85 @@
+##! Experimental features of the Cluster framework.
+
+@load base/frameworks/cluster
+
+module Cluster::Experimental;
+
+export {
+	## When using broker-enabled cluster framework, this event will be sent to
+	## the manager once a cluster node has set up all its outgoing connections
```

Comment:
A nit-picky suggestion here — it's not just that the node has set up outgoing connections (that to me would be `Broker::peer()`, but these have also checked in via `Cluster::node_up()`.